src/main/java/frc/robot/Robot.java

```diff
@@ -279,28 +305,47 @@ public void teleopSetup() {
     elevator.start();
     shooter.start();
     ballSubsystem.start();
-  }
 
+
+  }
+  TalonSRX turretMotor = new TalonSRX(RobotMap.TURRET_PORT);
+  TalonFX shooterMotor = new TalonFX(RobotMap.SHOOTER_PORT);
   /** This function is called periodically during operator control. */
   @Override
   public void teleopRoutine() {
-      double forward = this.driverPad.getValue(ControllerBindings.RIGHT_STICK, ControllerBindings.Y_AXIS);
-      double turn = this.driverPad.getValue(ControllerBindings.LEFT_STICK, ControllerBindings.X_AXIS);
+      double forward, turn;
+
+
+      forward = -this.operatorPad.getValue(ControllerBindings.LEFT_STICK, ControllerBindings.Y_AXIS);
+      turn = this.operatorPad.getValue(ControllerBindings.RIGHT_STICK, ControllerBindings.X_AXIS);
+
+      
       this.drive.drive(forward, turn);
 
       DpadDirection dpadDirection = angleToDpadDirection(this.operatorPad.getDpad());
+      boolean lastDpad = false; 
```

Comment:
What's the point of this variable?

Reply:
not sure to be honest, Massimo was using it for manual elevator control

Reply:
Please remove it, it currently does nothing

Reply:
removed